src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBarracksTower.java

```diff
@@ -11,51 +11,60 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
- * Building class of the BarracksTower.
+ * Building class for the Barracks Tower.
  */
+@SuppressWarnings("squid:MaximumInheritanceDepth")
 public class BuildingBarracksTower extends AbstractBuildingGuards
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2160)